Source/FolioReaderCenter.swift

```diff
@@ -772,6 +761,23 @@ public class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UICo
         }
         return nil
     }
+
+	// MARK: Public page methods
+
+	public func changePageWith(page page: Int, animated: Bool = false, completion: (() -> Void)? = nil) {
+		if page > 0 && page-1 < totalPages {
+			let indexPath = NSIndexPath(forRow: page-1, inSection: 0)
+			changePageWith(indexPath: indexPath, animated: animated, completion: { () -> Void in
+				self.updateCurrentPage({ () -> Void in
+					completion?()
+				})
+			})
+		}
+	}
+
+	public func getCurrentPage() -> FolioReaderPage? {
```

Comment:
I think is better to make `currentPage` public get with a private set, so we don't need it.
```swift
public private(set) var currentPage: FolioReaderPage?
```
And use it
```swift
FolioReader.sharedInstance.readerCenter.currentPage
```

What do you think?